holm/module_options/_actions.py

```
@@ -0,0 +1,325 @@
+            return func
+
+        return decorator
+
+    def _register_action(
+        self,
+        action: ActionDependency,
+        *,
+        with_layout: bool = False,
+        path: str | None,
+        tags: list[str | Enum] | None,
+        **route_args: Any,
```

Ruff (0.14.1)

Dynamically typed expressions (typing.Any) are disallowed in `**route_args`

(ANN401)